packages/core/admin/server/src/validation/common-validators.ts

```diff
@@ -110,7 +110,7 @@ export const permission = yup
         // @ts-expect-error yup types
         const action = getActionFromProvider(this.options.parent.action);
 
-        if (!action) {
+        if (!action || !subject) {
```

Comment:
was this a bug ?

Reply:
It was actually just a TS issue, but adding the initial check might just introduce a bug, I've reworked it